graphlearn_torch/python/partition/base.py

```diff
@@ -116,6 +116,58 @@ def save_feature_partition(
     torch.save(feature_partition.cache_feats, os.path.join(subdir, 'cache_feats.pt'))
     torch.save(feature_partition.cache_ids, os.path.join(subdir, 'cache_ids.pt'))
 
+def save_feature_partition(
+  output_dir: str,
+  partition_idx: int,
+  feature_partition: FeaturePartitionData,
+  group: str = 'node_feat',
+  graph_type: Optional[Union[NodeType, EdgeType]] = None
+):
+  r""" Save a feature partition into the output directory.
+  """
+  subdir = os.path.join(output_dir, f'part{partition_idx}', group)
+  if graph_type is not None:
+    subdir = os.path.join(subdir, as_str(graph_type))
+  ensure_dir(subdir)
+  append_tensor_to_file(os.path.join(subdir, 'feats.pkl'), feature_partition.feats)
+  append_tensor_to_file(os.path.join(subdir,'ids.pkl'), feature_partition.ids)
+  if feature_partition.cache_feats is not None:
+    torch.save(feature_partition.cache_feats, os.path.join(subdir, 'cache_feats.pt'))
+    torch.save(feature_partition.cache_ids, os.path.join(subdir, 'cache_ids.pt'))
+
+def save_partition_feature_chunk(
```

Comment:
Should we use `save_feature_partition_chunk` to keep the same naming style?

Reply:
renamed in the same style